tests/test_generics.py

```diff
@@ -104,10 +104,19 @@ def test_post_root_view(self):
         with self.assertNumQueries(1):
             response = self.view(request).render()
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertEqual(response.data, {'id': 4, 'text': 'foobar'})
+        self.assertEqual(response.data, {'id': 4, 'text': 'foobar', 'number': None})
         created = self.objects.get(id=4)
         self.assertEqual(created.text, 'foobar')
 
+    def test_post_null_integer_as_form_data(self):
+        """
+        None should resolve when a model's field has null=True.
+        """
+        data = {'text': 'foobar', 'number': None}
+        request = factory.post('/', data)
+        response = self.view(request).render()
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
Fair enough as a proof-of-status, but we'd want to refactor the test (for a lower footprint) if we did pull something in.